includes/admin/class-wcs-admin-meta-boxes.php

```diff
@@ -78,14 +78,15 @@ public function __construct() {
 	 * @param WP_Post|WC_Order|null $post_or_order_object The post or order currently being edited.
 	 */
 	public function add_meta_boxes( $post_type = '', $post_or_order_object = null ) {
+		$subscriptions_screen_id = wcs_is_custom_order_tables_usage_enabled() ? wc_get_page_screen_id( 'shop_subscription' ) : 'shop_subscription';
```

Comment:
Small nit-pick but the `wcs_is_custom_order_tables_usage_enabled()` is called multiple times in this function so we should pull it out into it's own variable

Reply:
Actually, we should check when `wc_get_page_screen_id()` was introduced instead of using `wcs_is_custom_order_tables_usage_enabled()`.

This will help later on with cleaning out outdated code once we remove support for WC x.x

Reply:
I've introduced a compat wrapper instead. See [cb67744](https://github.com/Automattic/woocommerce-subscriptions-core/pull/297/commits/cb67744512065b194d732d10cd7273813dbb2d77) and [c223624](https://github.com/Automattic/woocommerce-subscriptions-core/pull/297/commits/c22362481c344dc7c4e2c389d9f20060005747f9)